Document-Processing/Word/Word-Processor/angular/import.md

```diff
@@ -151,6 +151,50 @@ Here’s how to handle the server-side action for converting word document in to
 
 To know about server-side action, please refer this [page](./web-services-overview).
 
+## Enable Async (Lazy) Loading in Document Editor
+
+Document Editor provides an option to enable Async (Lazy) Loading using the [`openAsyncSettings`](https://ej2.syncfusion.com/angular/documentation/api/document-editor/openasyncsettings) property.
```

Comment:
Document Editor provides an option to load a document asynchronously using the...